src/Fields/integrals.jl

```diff
@@ -0,0 +1,40 @@
+#####
+##### Column integrals (definite)
+#####
+
+##### Allocating interface:
+column_∫_definite(field::CenterExtrudedFiniteDifferenceField) =
+    column_∫_definite(field, 1)
+column_∫_definite(field::FaceExtrudedFiniteDifferenceField) =
+    column_∫_definite(field, PlusHalf(1))
+
+column_∫_definite(field::Field, one_index::Union{Int, PlusHalf}) =
+    column_∫_definite(similar(level(field, one_index)), field)
+
+##### Non-allocating interface:
+function column_∫_definite!(col∫field::Field, field::Field)
+    bycolumn(axes(field)) do colidx
+        column_∫_definite!(col∫field[colidx], field[colidx])
+        nothing
+    end
+    return nothing
+end
+
+function column_∫_definite!(
+    col∫field::Field{<:DataLayouts.DataF},
+    field::Field{<:DataLayouts.VF},
+)
+    @inbounds col∫field[] = column_∫_definite(field)
+    return nothing
+end
+
+function column_∫_definite(field::Field{<:DataLayouts.VF})
+    Δz = Spaces.get_dz(axes(field))
+    ∫field = zero(eltype(field))
+    @inbounds for j in eachindex(parent(field))
+        ∫field += parent(field)[j] * parent(Δz)[j]
+    end
+    return ∫field
```

Comment:
Try to avoid calling `parent`
```suggestion
    field_data = Fields.field_values(field)
    Δz_data = Spaces.dz_data(axes(field))
    Nv = size(parent(Δz_data), 5)
    ∫field = zero(eltype(field))
    @inbounds for j in 1:Nv
        ∫field += field_data[j] * Δz_data[j]
    end
    return ∫field
```

Reply:
This suggestion fails the convergence test

```
  Expression: 2 < sum(abs.(cr)) / length(cr) < 2.01
   Evaluated: 2 < 0.09232257384130037 < 2.01
```

It looks like `Nv = size(parent(Δz_data), 5)` needs to be `Nv = size(parent(Δz_data), 1)`

Reply:
(this is after `bycolumn` is called on the field)